src/main/kotlin/org/rust/ide/actions/RsShowControlFlowGraphAction.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.actions
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.application.PathManager
+import com.intellij.openapi.ui.Messages
+import com.intellij.psi.util.parentOfType
+import com.intellij.util.io.createDirectories
+import com.intellij.util.io.delete
+import org.rust.lang.core.cfg.ControlFlowGraph
+import org.rust.lang.core.psi.RsBlock
+import org.rust.lang.core.psi.RsFunction
+import java.awt.Desktop
+import java.io.File
+import java.io.IOException
+import java.nio.file.Paths
+
+
+class RsShowControlFlowGraphAction : AnAction() {
+    private val CONTROL_FLOW_GRAPH = "control_flow_graph"
+    private val tempDir = Paths.get(PathManager.getTempPath()).resolve(CONTROL_FLOW_GRAPH)
```

Comment:
Are we sure that we want to create such files in tmp directory?
It's rather hard and inconvenient to find them there that can be important for such internal actions (for example, for bug fixing or something else)
Maybe it's better to create them in a folder in project base dir?
@vlad20012 what do you think?